techniques/general/hidden-control-3.html

```diff
@@ -17,10 +17,10 @@ <h2>Applicability</h2>
 		</section>
 		<section id="description">
 			<h2>Description</h2>
-			<p>The objective of this technique is to provide the information needed to identify a control when all contents of a larger component are editable. This primarily applies to a large design canvans with multiple editable components with it.</p>
+			<p>The objective of this technique is to provide the information needed to identify a control when all contents of a larger component are editable. This primarily applies to a large design canvas with multiple editable components with it.</p>
 			<p>To meet this technique, add text that:
 			<ul><li> is programatically and visually adjacent to the component, and</li>
```

Comment:
while not directly related to this change, while you're here, might be worth removing the redundant extra space at the start/end of the `<li>`s